src/pages/distroSettings/tabs/HostTab/getFormSchema.tsx

```diff
@@ -1,24 +1,58 @@
-import { InlineCode } from "@leafygreen-ui/typography";
 import { GetFormSchema } from "components/SpruceForm";
 import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
+import { Arch, BootstrapMethod, Provider, SshKey } from "gql/generated/types";
 import {
-  Arch,
-  BootstrapMethod,
-  CommunicationMethod,
-  FeedbackRule,
-  HostAllocatorVersion,
-  OverallocatedRule,
-  Provider,
-  RoundingRule,
-  SshKey,
-} from "gql/generated/types";
+  linuxArchitectures,
+  nonWindowsArchitectures,
+  windowsArchitectures,
+} from "./constants";
+import {
+  acceptableHostIdleTime,
+  arch,
+  authorizedKeysFile,
+  bootstrapMethod,
+  clientDir,
+  communicationMethod,
+  env,
+  feedbackRule,
+  futureHostFraction,
+  homeVolumeFormatCommand,
+  hostsOverallocatedRule,
+  icecreamConfigPath,
+  icecreamSchedulerHost,
+  isVirtualWorkStation,
+  jasperBinaryDir,
+  jasperCredentialsPath,
+  lockedMemoryKb,
+  maximumHosts,
+  minimumHosts,
+  numFiles,
+  numProcesses,
```

Comment:
For some parts of the form, I think it could help to group these fields as the section and then just import that object, so that you don't have to import a bunch of fields.

e.g. like having a `{ schema: <stuff>, uiSchema: <stuff> }` object for the whole allocation section:

<img width="433" alt="Screenshot 2023-09-27 at 12 13 45 PM" src="https://github.com/evergreen-ci/spruce/assets/47064971/9bb9bc13-8a97-4a53-a3f2-a7e928f5f562">